### PR TITLE
New Feature -- custom product page

### PR DESCRIPTION
**Pull request by mammadjafar**
Possibility to add custom page for product, in order to add custom page:
1. create hbs file in theme folder
2. add new productView in  setting file
3. Select it from admin panel

Note: there is some coding still needed like implementing it in update page some validation etc.

### CODE REVIEW

**Comment by mammadjafar** [comment]
@mrvautin should I make an issue about this feature ?

**Comment by mrvautin** [comment]
Could you rebase this onto the current master? Also can see there are some linting issues. Can you also setup linting and ensure there are no issues.

**Comment by mammadjafar** [comment]
@mrvautin Hi, can't find travis logs what is failing, any hints ?

**Comment by mammadjafar** [comment]
@mrvautin any comment/help/tips ?

File: config/settings.json [hunk 1]
```diff
@@ -10,21 +10,29 @@
...
+    "maxQuantity": 25,
+    "twitterHandle": "",
+    "facebookAppId": "",
+    "modules": {
+        "enabled": {
+            "shipping": "shipping-basic",
+            "discount": "discount-voucher"
+        }
+    },
+    "productViews": [{"key": "default", "value": "Defaut"}]
```
Comment (mrvautin) [inline config/settings.json:37]: `Default` typo

**Comment by mrvautin** [comment]
It looks like you have made changes to the `settings.json` which is now failing the schema validation.

File: config/baseSchema.json [hunk 2]
```diff
@@ -119,6 +113,37 @@
...
+                }
+            },
+            "required": ["enabled"],
+            "default": {
+                "enabled": {
+                    "shipping": "shipping-basic"
+                }
+            },
+        "productViews": {
+            "type": "array"
```
Comment (mammadjafar) [inline config/baseSchema.json:146]: @mrvautin you mentioned change in settings.js and validation, isn't this part correct ?

Comment (mrvautin) [reply config/baseSchema.json:146]: Not sure I haven’t had a chance to check but the Travis tests are failing. Can you run the tests and find the error?

**Comment by KingRoyaltea** [comment]
Heyy, i tried running the original one,  i'm getting an something about Missing "START" script

**Pull request closed without merging**